protos/FunctionRpc.proto

```diff
@@ -251,6 +251,6 @@ message RpcHttp {
   map<string,string> params = 10;
   string status_code = 12;
   map<string,string> query = 15;
-  bool is_raw = 16;
+  bool enableContentNegotiation = 16;
```

Comment:
Proto file is moved here https://github.com/Azure/azure-functions-language-worker-protobuf

Reply:
Really need to remove all the duplicates